src/ray/common/id.h

```diff
@@ -376,6 +333,47 @@ class PlacementGroupID : public BaseID<PlacementGroupID> {
 
 typedef std::pair<PlacementGroupID, int64_t> BundleID;
 
+class LeaseID : public BaseID<LeaseID> {
+ private:
+  static constexpr size_t kUniqueBytesLength = 4;
+
+ public:
+  static constexpr size_t kLength = kUniqueBytesLength + kUniqueIDSize;
+
+  static constexpr size_t Size() { return kLength; }
+
+  /// Creates a `LeaseID` from a specific worker ID.
+  ///
+  /// \param worker_id The worker id to which this lease belongs.
+  ///
+  /// \return The `LeaseID` for the worker lease.
+  static LeaseID FromWorkerId(const WorkerID &worker_id);
+
+  /// Creates a `LeaseID` from a driver ID. The counter bits are nulled out only for
+  /// driver as we need a predetermined lease value that can be calculated indepently by
+  /// the raylet without having to send the ID over.
```

Comment:
so is there only one static ID for every driver? what is it used for?

If I'm understanding correctly, maybe we should call it something really dumb obvious like `DriverSingletonLeaseID`

Reply:
or `DriverBootstrapLeaseID` (if it's used for some bootstrapping)